docs/developer-guide/ksqldb-reference/select-push-query.md

```diff
@@ -215,6 +218,27 @@ the order was placed, and shipped within 2 hours of the payment being received.
         INNER JOIN shipments s WITHIN 2 HOURS ON s.id = o.id;
 ```
 
+The GRACE PERIOD, part of the WITHIN clause, allows the join to process late records for up

```

Comment:
`late` -> `out-of-order` 

"late" record are by definition of the term _late_ (as we use it) after the grace period and thus are dropped